src/StrList.cc

```diff
@@ -49,6 +49,21 @@ strListIsMember(const String * list, const SBuf &m, char del)
     return 0;
 }
 
+bool
+strListIsMember_if(const String *list, char del, MemberFilter filter)
+{
+    const char *pos = NULL;
```

Comment:
```suggestion
    const char *pos = nullptr;
```
Please check for other NULL occurrences.

Reply:
commit e471cbdb